### PR TITLE
EVM Smart Contract Support

### LINKED ISSUE #4818: [Feature Request] : update the ecosystem projects supporting ink contracts and evm smart contracts
https://wiki.polkadot.network/docs/build-smart-contracts#parity-tech

### PR DESCRIPTION
**Pull request by CrackTheCode016**
Closes https://github.com/w3f/polkadot-wiki/issues/4818

---
Adds clarification for whether a parachain supports EVM or ink!

### CODE REVIEW

File: docs/build/build-smart-contracts.md [hunk 1]
```diff
@@ -12,6 +12,9 @@ blockchain, does not support smart contracts natively. However, parachains which
 blockchains on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} are equipped with the
 functionality to support smart contracts.
 
+The two primary, supported smart contract environments that [ink!](#ink) and EVM. There are multiple
+[parachains that support both environments](#parachains).
```
Comment (profradha) [inline docs/build/build-smart-contracts.md]:
```suggestion
The two primary supported smart contract environments are [ink!](#ink) and EVM. There are multiple
[parachains that support both environments](#parachains).
```

File: docs/build/build-smart-contracts.md [hunk 2]
```diff
@@ -191,6 +194,9 @@ created and maintained list of different smart contract platforms building on
 
 #### Moonbeam
 
+- ink! Support: **Unsupported**
```
Comment (profradha) [inline docs/build/build-smart-contracts.md]:
```suggestion
- ink! : **Unsupported**
```

File: docs/build/build-smart-contracts.md [hunk 3]
```diff
@@ -191,6 +194,9 @@ created and maintained list of different smart contract platforms building on
...
+- EVM (Solidity) Support: [**Supported**](https://moonbeam.network/networks/moonbeam/)
```
Comment (profradha) [inline docs/build/build-smart-contracts.md]:
```suggestion
- EVM (Solidity) : [**Supported**](https://moonbeam.network/networks/moonbeam/)
```

File: docs/build/build-smart-contracts.md [hunk 4]
```diff
@@ -207,6 +213,10 @@ Try deploying a smart contract to Moonbeam by following their
 
 #### Astar
 
+- ink!/Wasm Support: [**Supported**](https://docs.astar.network/docs/build/#wasm-smart-contracts)
```
Comment (profradha) [inline docs/build/build-smart-contracts.md]:
```suggestion
- ink!/Wasm: [**Supported**](https://docs.astar.network/docs/build/#wasm-smart-contracts)
```

File: docs/build/build-smart-contracts.md [hunk 5]
```diff
@@ -207,6 +213,10 @@ Try deploying a smart contract to Moonbeam by following their
...
+- EVM (Solidity)
+  Support:[ **Supported**](https://docs.astar.network/docs/build/#evm-smart-contracts)
```
Comment (profradha) [inline docs/build/build-smart-contracts.md]:
```suggestion
- EVM (Solidity): [ **Supported**](https://docs.astar.network/docs/build/#evm-smart-contracts)
```

File: docs/build/build-smart-contracts.md [hunk 6]
```diff
@@ -223,6 +233,9 @@ Try deploying an Ethereum or ink! smart contract by following their
 
 #### Acala
 
+- ink! Support: **Unsupported**
```
Comment (profradha) [inline docs/build/build-smart-contracts.md]:
```suggestion
- ink!: **Unsupported**
```

File: docs/build/build-smart-contracts.md [hunk 7]
```diff
@@ -223,6 +233,9 @@ Try deploying an Ethereum or ink! smart contract by following their
...
+- EVM (Solidity) Support: [**Supported**](https://wiki.acala.network/build/development-guide)
```
Comment (profradha) [inline docs/build/build-smart-contracts.md]:
```suggestion
- EVM (Solidity): [**Supported**](https://wiki.acala.network/build/development-guide)
```

File: docs/build/build-smart-contracts.md [hunk 8]
```diff
@@ -234,6 +247,10 @@ Try deploying an Acala EVM smart contract by following their
 
 #### Phala
 
+- ink! Support: **Unsupported**
```
Comment (profradha) [inline docs/build/build-smart-contracts.md]:
```suggestion
- ink!: **Unsupported**
```

File: docs/build/build-smart-contracts.md [hunk 9]
```diff
@@ -234,6 +247,10 @@ Try deploying an Acala EVM smart contract by following their
...
+- EVM (Solidity) Support: **Unsupported**
```
Comment (profradha) [inline docs/build/build-smart-contracts.md]:
```suggestion
- EVM (Solidity): **Unsupported**
```

File: docs/build/build-smart-contracts.md [hunk 10]
```diff
@@ -245,6 +262,10 @@ Try deploying a confidential smart contract by following their
 
 #### Darwinia
 
+- ink! Support: **Unsupported**
```
Comment (profradha) [inline docs/build/build-smart-contracts.md]:
```suggestion
- ink!: **Unsupported**
```

**Review by DrW3RK — APPROVED** [review]
Looks good! Just removed some redundant text.

**Comment by CrackTheCode016** [comment]
Thank you!

**Pull request merged**